contracts/testing/test_request.go

```diff
@@ -0,0 +1,10 @@
+package testing
+
+type TestRequest interface {
+	WithHeaders(headers map[string]string) TestRequest
+	WithHeader(key, value string) TestRequest
+	WithoutHeader(key string) TestRequest
+	WithCookies(cookies map[string]string) TestRequest
```

Comment:
```suggestion
	WithCookies(cookies map[string]any) TestRequest
```

Reply:
I am going to cast it to string(http.Cookie expects value of type string) anyways so it's better to be done by developers